lib/code_corps/model/conversation_part.ex

```diff
@@ -19,6 +19,7 @@ defmodule CodeCorps.ConversationPart do
   schema "conversation_parts" do
     field :body, :string, null: false
     field :read_at, :utc_datetime, null: true
+    field :part_type, :string
```

Comment:
I would make this `default: "comment"` so it doesn't have to be specified for most cases.

Reply:
How does it get set to any other type?  If I set it to a default, I can figure out a way to test the part_type since it doesn't get included in the changeset...